### DISCUSSION TITLE
How do i enable MIRV missiles

### DISCUSSION THREAD

**Discussion by P3tux0** [HbmMods/Hbm-s-Nuclear-Tech-GIT#651] [Q&A]
How do i enable MIRV missile and other blacklisted items?

Sorce:
https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/blob/master/src/main/java/com/hbm/entity/missile/EntityMissileMirv.java

How do i enable it

**Comment by alex-s168** [comment]
I don't think it is finished because there is an unmerged pull request that adds them (don't blame me if I'm wrong)
https://github.com/HbmMods/Hbm-s-Nuclear-Tech-GIT/pull/672
